jdaviz/core/astrowidgets_api.py

```diff
@@ -196,19 +184,8 @@ def zoom_level(self, val):
         if val == 'fit':
             self.state.reset_limits()
             return
-        else:
-            cur_xcen = (self.state.x_min + self.state.x_max) * 0.5
-            new_dx = self.shape[1] * 0.5 / val
-            new_x_min = cur_xcen - new_dx
-            new_x_max = cur_xcen + new_dx
-
-        with delay_callback(self.state, 'x_min', 'x_max'):
-            self.state.x_min = new_x_min - 0.5
-            self.state.x_max = new_x_max - 0.5

```

Comment:
@pllim should this be `+ 0.5` instead (assuming the 0.5 is to account for the margin added around the data)?  Otherwise, where is this coming from?

Reply:
I don't quite remember either and https://github.com/spacetelescope/jdaviz/pull/744 has no record of it. I think it has something to do with user input being pixel center but viewer wants pixel corner. Not 100% sure.

Reply:
I was sure I worked out the math properly though using API + mouse cursor info. Does it not work anymore?

Reply:
I was seeing wandering when changing the zoom-level and switching to `+` seemed to fix it, but maybe introduced that somewhere else.  I'll go back to main and see if I can trigger the same there or not.  Either way, doesn't _seem_ to be the culprit for the failing test.